packages/loader/driver-utils/src/messageRecognition.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IDocumentMessage, ISequencedDocumentMessage, MessageType } from "@fluidframework/protocol-definitions";
+
+export function isClientMessage(message: ISequencedDocumentMessage | IDocumentMessage) {
+    switch (message.type) {
```

Comment:
I'd prefer for this function to explicitly include call to isRuntimeMessage() and return true if it returned true.
Such that if we ever add new op type runtime message, this function automatically includes it.

Might be useful to add an assert somewhere (in DeltaManager) to enforce such inclusion
assert(!isRuntimeMessage(op) || isClientMessage)op));